src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -0,0 +1,219 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Net.Sockets;
+using System.Net.Test.Common;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Security.Tests
+{
+    public class ServerAsyncAuthenticateTest
+    {
+        private readonly ITestOutputHelper _log;
+        private readonly X509Certificate2 _serverCertificate;
+
+        public ServerAsyncAuthenticateTest()
+        {
+            _log = TestLogging.GetInstance();
```

Comment:
What is the difference between using `TestLogging.GetInstance()` and using the constructor overload which passes in an `ITestOutputHelper` instance?  We have other tests that use the latter.